cmd/container-use/main_test.go

```diff
@@ -0,0 +1,60 @@
+package main_test
+
+import (
+	"log/slog"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
the pr description explained why all this faff was necessary on windows, but do we really need to fuck with it on unixes too?

(implicitly, couldn't this be done with build flags?)